educational-core/src/com/jetbrains/edu/learning/codeforces/courseFormat/CodeforcesCourse.kt

```diff
@@ -39,7 +39,7 @@ class CodeforcesCourse : Course {
   override fun isViewAsEducatorEnabled(): Boolean = false
 
   fun getContestUrl(): String = getContestURLFromID(id)
-  fun getSubmissionUrl(): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode"
+  fun getSubmissionUrl(task: CodeforcesTask? = null): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode${if (task == null) "" else "&submittedProblemIndex=" + task.name.substringBefore(".")}"
```

Comment:
I don't see why the task could be nullable here, could you explain, please?

Reply:
Just to keep old semantic if it need.